drivers/barometer.cpp

```diff
@@ -48,25 +48,34 @@ extern "C"
 #include "util/maths.h"
 }
 
-Barometer::Barometer() :
-    has_been_calibrated_(false)
-{
+// Definition of static member
+float Barometer::pressure_at_sea_level_;
```

Comment:
Same as `INS::origin_`. This is static, so the variable needs to be defined somewhere